Causely Integration update (ECOINT-82) (#2591)
DataDog/integrations-extras: causely/README.md

## Overview

> The [Causal AI platform][1] supercharges your observability stack by applying its built-in intelligence to your telemetry data. It enables self-managed, resilient applications by bridging observability with automated orchestration. These analytics serve as a real-time blueprint for resolving incidents and preventing them in the first place.

Author: I can drop the blueprint part

> Causely delivers instant value with no complex setup or configuration. It helps shift your organization from a reactive to a proactive mode of operational. Instead of flood of alerts, Causely tells you exactly what you need to know about all active and emerging application failures. With Causely, understand why a failure happened, what the root cause was, and how to fix it.

Reviewer: Should this be "mode of operational" or "mode of operations"?

Author: thank you!

> Causely delivers value instantly without complex setup or configuration. The system is designed to help you shift your organization from a reactive mode to a proactive mode of operations.  Instead of flood of alerts, Causely tells you exactly what you need to know about all active and emerging application failures. With Causely, understand why a failure happened, what the root cause was, and how to fix it.

Reviewer: ```suggestion
Causely delivers instant value with no complex setup or configuration. It helps shift your organization from a reactive to a proactive operational mode..  Instead of flood of alerts, Causely tells you exactly what you need to know about all active and emerging application failures. With Causely, understand why a failure happened, what the root cause was, and how to fix it.
```

Author: thanks
